### ISSUE TITLE
Exclude QuarkusTest from surefire executions

### ISSUE THREAD

**Issue by pjgg** [quarkus-qe/quarkus-test-suite#597]
All Quarkus test should ends with *Test and should be excluded from Openshift/surefire executions

**Comment by michalvavrik** [comment]
There is one test that needs to be renamed `BouncyCastleFipsJsseIT`

```
[mvavrik@fedora quarkus-test-suite]$ grep -l 'io.quarkus.test.junit.QuarkusTest' . -R | sort | uniq
./http/http-minimum-reactive/src/test/resources/HelloResourceTest.java.template
./http/http-minimum-reactive/src/test/resources/NoFunctionalTest.java.template
./http/http-minimum-reactive/target/test-classes/HelloResourceTest.java.template
./http/http-minimum-reactive/target/test-classes/NoFunctionalTest.java.template
./http/http-minimum/src/test/resources/HelloResourceTest.java.template
./http/http-minimum/src/test/resources/NoFunctionalTest.java.template
./http/http-minimum/target/test-classes/HelloResourceTest.java.template
./http/http-minimum/target/test-classes/NoFunctionalTest.java.template
./http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/PureVertxHttpClientTest.java
./http/vertx-web-client/target/test-classes/io/quarkus/ts/http/vertx/webclient/PureVertxHttpClientTest.class
./lifecycle-application/src/test/java/io/quarkus/ts/lifecycle/AlphabeticallyFirstTest.java
./lifecycle-application/target/test-classes/io/quarkus/ts/lifecycle/AlphabeticallyFirstTest.class
./oc-rep/secret-reproducer/target/test-classes/org/acme/GreetingResourceTest.class
./quarkus-extension-abc/integration-tests/target/test-classes/io/quarkiverse/extension/abc/it/ExtensionAbcResourceTest.class
./security/basic/src/test/java/io/quarkus/ts/openshift/security/basic/SecurityTest.java
./security/basic/src/test/java/io/quarkus/ts/openshift/security/basic/SecurityTestProfile.java
./security/basic/target/test-classes/io/quarkus/ts/openshift/security/basic/SecurityTest.class
./security/basic/target/test-classes/io/quarkus/ts/openshift/security/basic/SecurityTestProfile.class
./security/bouncycastle-fips/bcFipsJsse/src/test/java/io/quarkus/ts/security/bouncycastle/fips/jsse/BouncyCastleFipsJsseIT.java
./security/bouncycastle-fips/bcFipsJsse/target/test-classes/io/quarkus/ts/security/bouncycastle/fips/jsse/BouncyCastleFipsJsseIT.class
./spring/spring-data/src/test/java/io/quarkus/ts/spring/data/di/SpringDiTest.java
./spring/spring-data/target/test-classes/io/quarkus/ts/spring/data/di/SpringDiTest.class
./sql-db/panache-flyway/src/test/java/io/quarkus/ts/sqldb/panacheflyway/dbpool/AgroalPoolTest.java
./sql-db/panache-flyway/src/test/java/io/quarkus/ts/sqldb/panacheflyway/dbpool/AgroalTestProfile.java
./sql-db/panache-flyway/target/test-classes/io/quarkus/ts/sqldb/panacheflyway/dbpool/AgroalPoolTest.class
./sql-db/panache-flyway/target/test-classes/io/quarkus/ts/sqldb/panacheflyway/dbpool/AgroalTestProfile.class
./sql-db/vertx-sql/src/test/java/io/quarkus/ts/vertx/sql/dbpool/PostgresPoolTest.java
./sql-db/vertx-sql/src/test/java/io/quarkus/ts/vertx/sql/dbpool/PostgresqlTestProfile.java
./sql-db/vertx-sql/target/test-classes/io/quarkus/ts/vertx/sql/dbpool/PostgresPoolTest.class
./sql-db/vertx-sql/target/test-classes/io/quarkus/ts/vertx/sql/dbpool/PostgresqlTestProfile.class
```

**Comment by jedla97** [comment]
This seems to be resolved by #1230 so I'm closing it.
